controllers/util/drpolicy_util.go

```diff
@@ -22,6 +22,10 @@ func DrpolicyClusterNames(drpolicy *rmn.DRPolicy) []string {
 	return drpolicy.Spec.DRClusters
 }
 
+func DrpolicyClusterNamesAsASet(drpolicy *rmn.DRPolicy) sets.String {
```

Comment:
(nit) we should change he name of the function to:
`DRPolicyClusterNamesAsASet`. Same for `DRPolicyClusterNames`
The first 3 letters capitalized.

Reply:
done